tests/models/gemma2/test_modeling_gemma2.py

```diff
@@ -58,28 +58,12 @@ class Gemma2ModelTester(CausalLMModelTester):
         config_class = Gemma2Config
         base_model_class = Gemma2Model
         causal_lm_class = Gemma2ForCausalLM
-        sequence_class = Gemma2ForSequenceClassification
-        token_class = Gemma2ForTokenClassification
-    pipeline_model_mapping = (
```

Comment:
`pipeline_model_mapping` does nothing in `XXXModelTester`, it should be set in `XXXModelTest`. Possibly a bad copy-paste?

(this pattern occurs in other places as well)